_requirements/test.txt

```diff
@@ -1,3 +1,4 @@
+httpx>=0.27.0
```

Comment:
it is Feb 21, 2024 so not to new...
https://pypi.org/project/httpx/0.27.0/

Reply:
Yeah, moved it to test dependency and removed from package dependency so we should be safe.